_posts/2019-10-24-this-week-in-glean.md

```diff
@@ -0,0 +1,60 @@
+---
+permalink: "/{{ year }}/{{ month }}/{{ day }}/this-week-in-glean"
+title: "This Week in Glean: A Release"
+published_date: "2019-10-24 17:30:00 +0200"
+layout: post.liquid
+data:
+  route: blog
+  tags:
+    - mozilla
+    - rust
+---
+
+(“This Week in Glean” is a series of blog posts that the Glean Team at Mozilla is using to try to communicate better about our work. They could be release notes, documentation, hopes, dreams, or whatever: so long as it is inspired by Glean.)
+
+Last week's blog post: [This Week in Glean: Glean on Desktop (Project FOG)](https://chuttenblog.wordpress.com/2019/10/17/this-week-in-glean-glean-on-desktop-project-fog/) by chutten.
+
+---
+
+Back in June when [Firefox Preview shipped](https://blog.mozilla.org/futurereleases/2019/06/27/reinventing-firefox-for-android-a-preview/), it also shipped with Glean, our new solution for Telemetry on mobile.
+Georg recently blogged about the design principles of Glean in [Introducing Glean — Telemetry for humans](https://medium.com/georg-fritzsche/introducing-glean-telemetry-for-humans-4e8b4788b8ad).
+
+Plans for improving mobile telemetry for Mozilla go back as as far as December 2017.
+The first implementation of Glean was started around August 2018, all written in Kotlin (though back then it was mostly ideas in a bunch of text documents).
+This implementation shipped in Firefox Preview and was used up until now.
+
+On March 18th I [created an initial Rust workspace](https://github.com/mozilla/glean/commit/95b6bcc03616c8d7c3e3e64e99ee9953aa06a474).
+This kicked of a rewrite of Glean using Rust to become a cross-platform telemetry SDK to be used on Android, iOS and eventually coming back to desktop platforms again.
+
+1382 commits later[^1] I tagged [v19.0.0](https://github.com/mozilla/glean/commit/1ac00bf63daea97f6e2d6fa36980279eecf5a800)[^2].
+
+Obviously that doesn't make people use it right away, but given all all consumers of Glean right now are Mozilla products, it's up on us to get them to use it.
+So [Alessio did just that](https://github.com/mozilla-mobile/android-components/pull/4620) by upgrading Android Components, a collection of Android libraries to build browsers or browser-like applications, to this new version.
+
+This will soon roll out to nightly releases of Firefox Preview and, given we don't hit any larger bugs, hit the release channel in about 2 weeks.
+Additionally, that finally unblocks the Glean team to work on new features, ironing out some sharp edges and bringing Glean to Firefox on Desktop.
+Oh, and of course we still need to actually release it for iOS.
+
+## Thanks
+
+Glean in Rust is the project I've been constantly working on since March.
+But getting it to a release was a team effort with help from a multitude of people and teams.
+
+Thanks to everyone on the Glean team:
+
+* [Alessio Placitelli](https://github.com/dexterp37/)
```

Comment:
It's missing Bea :) Even though she recently joined, it might be nice to call her out ;)